doc/source/ray-air/examples/automl_with_ray_air.ipynb

```diff
@@ -0,0 +1,858 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "596e2463",
+   "metadata": {},
+   "source": [
+    "# Simple AutoML for time series with Ray AIR"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "676f0ada",
+   "metadata": {},
+   "source": [
+    "AutoML (Automatic Machine Learning) boils down to picking the best model for a given task and dataset. In {doc}`this Ray Core example </ray-core/examples/automl_for_time_series>`, we showed how to build an AutoML (or more precisely, an AutoTS) system which will chooses the best `statsforecast` model and its corresponding hyperparameters for a time series regression task on the M5 dataset.\n",
+    "\n",
+    "The basic steps were:\n",
+    "\n",
+    "1. Define a set of autoregressive forecasting models to search over. For each model type, we also define a set of model parameters to search over.\n",
+    "2. Perform temporal cross-validation on each model and model parameter configuration in parallel.\n",
+    "3. Pick the best performing model as the output of the AutoML system.\n",
+    "\n",
+    "We see that these steps fit into the framework of a hyperparameter optimization problem that can be tackled with the [Ray AIR Tuner](air-tuner)!\n",
+    "\n",
+    "In this notebook, we will show how to create an AutoML system with Ray AIR. Compared to using Ray Core, we will leverage the higher-level Tuner API to define the model and hyperparameter search space, parallelize cross-validation of different models, and finally analyze results to pull the best-performing model for the time-series dataset. Similar to the Ray Core example, we will be using only one partition of the M5 dataset for this AutoML example."
```

Comment:
can we link to the ray core example?

Reply:
Can you also break this up into bullet points?

```
In this notebook, we will show how to:
  1. Create an AutoML system for time series forecasting
  2. Leverage the ...